media-libs/jpeg/jpeg-9c.recipe

```diff
@@ -58,9 +58,7 @@ BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	"
 BUILD_PREREQUIRES="
-	cmd:aclocal
-	cmd:autoconf
-	cmd:automake
```

Comment:
`cmd:aclocal` and `cmd:automake` are both in the automake package, and the autoconf package (which currently provides `cmd:autoreconf` does not depend on automake, so I think the only `cmd:` you can safely remove here is `cmd:autoconf`.
In other words, I think you should keep `cmd:aclocal` and `cmd:automake`, so that `cmd:autoreconf` will find them.
(I haven't checked this yet, so I might be wrong.)